README.md

```diff
@@ -24,7 +24,7 @@ from node2vec import Node2Vec
 graph = nx.fast_gnp_random_graph(n=100, p=0.5)
 
 # Precompute probabilities and generate walks - **ON WINDOWS ONLY WORKS WITH workers=1**
-node2vec = Node2Vec(graph, dimensions=64, walk_length=30, num_walks=200, workers=4) 
+node2vec = Node2Vec(graph, dimensions=64, walk_length=30, num_walks=200, workers=4)  # pass temp_folder for the big graphs
```

Comment:
I do not get it how can I pass temp_folder in Node2Vec, can you please explain me? I have a graph that consist of 141386 nodes and 49296336 edges.

Reply:
Look at the README, it is explained there. Pass `temp_folder` to the `Node2Vec` constructor.

Reply:
temp_folder = ?

Reply:
It is the (string) path to the folder. This folder should be on the disk with enough memory to hold the data.

Reply:
I have written temp_folder = path of tmp_data folder but I got this error: __init__() got an unexpected keyword argument 'temp_folder'

Reply:
It shouldn't. [This](https://github.com/eliorc/node2vec/blob/master/node2vec/node2vec.py#L25) is the definition of `Node2Vec`. It accepts the `temp_folder` argument. Are you sure you are calling it right?

Reply:
I have written the following:
Node2Vec(graph, dimensions=64, walk_length=30, num_walks=200, workers=4, temp_folder="/home/my_work/data/tmp_data")

Reply:
1. Can you paste the stack trace?
2. Can you check the source code of Node2Vec?

Reply:
TypeError: __init__() got an unexpected keyword argument 'temp_folder'

Reply:
Thanks for your immediate reply. The problem has been solved after installing from the source.